app/src/main/java/com/github/houseorganizer/houseorganizer/LoginEmail.java

```diff
@@ -0,0 +1,81 @@
+package com.github.houseorganizer.houseorganizer;
+
+import android.annotation.SuppressLint;
+import android.content.Intent;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import com.google.firebase.auth.FirebaseAuth;
+
+public class LoginEmail extends AppCompatActivity implements View.OnClickListener {
+
+    private FirebaseAuth mAuth;
+
+    private final String TAG = "LoginEmail";
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login_email);
+
+        mAuth = FirebaseAuth.getInstance();
+
+        findViewById(R.id.log_email_signin_button).setOnClickListener(this);
+        findViewById(R.id.log_email_register_button).setOnClickListener(this);
+    }
+
+    @SuppressLint("NonConstantResourceId")
+    @Override
+    public void onClick(View v) {
+        switch (v.getId()) {
+            case R.id.log_email_signin_button:
+                if (inputsNotEmpty()) signInWithEmail(v);
+                break;
+            case R.id.log_email_register_button:
+                startActivity(new Intent(LoginEmail.this, RegisterEmail.class));
+                break;
+        }
+    }
```

Comment:
Same as onClick in LoginActivity, unless there is a reason to write it this way it is cleaner to do it either in onCreate or in the xml